pkg/config/application_config.go

```diff
@@ -26,4 +26,5 @@ type ApplicationConfiguration struct {
 	Relay             bool                    `yaml:"Relay"`
 	RPC               rpc.Config              `yaml:"RPC"`
 	UnlockWallet      Wallet                  `yaml:"UnlockWallet"`
+	Oracle            OracleConfiguration     `yaml:"Oracle"`
```

Comment:
The question is similar to https://github.com/nspcc-dev/neo-go/pull/1427#discussion_r564696678, don't we want to place all modules configuration inside `Modules`?

Reply:
I'm OK with `config.Oracle`. Or `config.OracleService`.